### PR TITLE
[16.0][MIG] account_due_list_days_overdue

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
@ForgeFlow

### CODE REVIEW

**Comment by mileo** [comment]
Could you reopen?

**Review by MiquelRForgeFlow — APPROVED** [review]

File: account_due_list_days_overdue/models/account_move_line.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    @api.model
+    def fields_view_get(
+        self, view_id=None, view_type="form", toolbar=False, submenu=False
+    ):
+        result = super().fields_view_get(
+            view_id,
+            view_type,
+            toolbar=toolbar,
+            submenu=submenu,
+        )
```
Comment (rvalyi) [inline account_due_list_days_overdue/models/account_move_line.py]: this fields_view_get method does not exist anymore in Odoo v16 models.py so I'm afraid this override won't be called and it needs to be migrated, Usually this is replaced by the new `def _get_view(self, view_id=None, view_type="form", **options):` in v16...

**Review by rvalyi — CHANGES_REQUESTED** [review]

File: account_due_list_days_overdue/models/account_move_line.py (hunk 1 above)
Comment (ChrisOForgeFlow) [reply account_due_list_days_overdue/models/account_move_line.py]: I changed to new function, I suppose use get_view instead _get_view function is not an issue in implementation

**Comment by JordiBForgeFlow** [comment]
@rvalyi I can see that you requested changes and they have been attended. Can you update your review? We're hoping to get this one merged.

**Comment by matteoopenf** [comment]
@rvalyi  any news?

**Review by rvalyi — APPROVED** [review]
sorry for the delay, I think the migration is OK now with the changes in view_get

**Comment by matteoopenf** [comment]
> sorry for the delay, I think the migration is OK now with the changes in view_get

Thank you

**Pull request merged**